oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -419,6 +419,11 @@ query:
   selector: ${SW_QUERY:graphql}
   graphql:
     path: ${SW_QUERY_GRAPHQL_PATH:/graphql}
+    # Enable the log testing API to test the LAL.
+    # NOTE: This API evaluates untrusted code on the OAP server.
```

Comment:
Could LAL engine disable class call? And limit to the classes we know?

Reply:
Theoretically it can, but there are a lot of workarounds to bypass and we won't be able to confidently enable it by default and this disclaimer is necessary in any point.

Reply:
We wouldn't enable it in default, but I hope we could provide security as much as possible, because I am planning to enable it on demo env

Reply:
> We wouldn't enable it in default, but I hope we could provide security as much as possible, because I am planning to enable it on demo env

I'd rather track this in a separate issue which might share with all groovy-based DSLs, including MAL.

Reply:
It is fine through another PR. I am just concerned before that, we could not use official demo env for @Fine0830 's dev.

Reply:
I'll manage to share a private env for her